android/assets/ui/languages.xml

```diff
@@ -9,6 +9,7 @@
     <Language id="ru" name="Русский" fontSet="xolonium"/>
     <Language id="bn" name="বাংলা" fontSet="noto-bengali"/>
     <Language id="zh_CN" name="简体中文" fontSet="noto-cjk"/>
+    <Language id="pt_BR" name="PortugueseBR" fontSet="xolonium-kwajong"/>	
```

Comment:
The text in the `name` attribute is the one shown in the language list. You can see it in this screenshot:
![screenshot-2022-09-10-100053 052](https://user-images.githubusercontent.com/3575/189474740-596deb7d-873e-4f9e-aaeb-f1578637e620.png)

This text should be written in the language of the translation itself, can you change it to say "Brazilian Portuguese" in Brazilian Portuguese?